.config/nextest.toml

```diff
@@ -0,0 +1,6 @@
+[profile.default]
+slow-timeout = { period = "60s", terminate-after = 5 }
```

Comment:
can we add this to the readme? if someone creates a test which takes time to execute, they should know to add it in the nextest config.

Reply:
I have added it to the `contributing.md`